.circleci/config.yml

```diff
@@ -329,9 +329,11 @@ commands:
               curl https://sh.rustup.rs -sSf -o rustup.sh
               chmod 755 ./rustup.sh
               ./rustup.sh -y --profile minimal --component clippy --component rustfmt --default-toolchain none
-              $HOME/.cargo/bin/rustc -V
             fi
-            echo 'export PATH="$HOME/.cargo/bin:$PATH"' >> "$BASH_ENV"
+            echo '. "$HOME/.cargo/env"' >> "$BASH_ENV"
```

Comment:
something isn't quite right yet it seems 🤔

Reply:
Alright, it took a few attempts but the toolchain is working on all platforms now

Reply:
let's merge!